src/Cli/Program.cs

```diff
@@ -123,18 +122,43 @@ public static int Main(string[] args)
 
                     if (options.GraphQL)
                     {
-                        Task.Delay(1000).Wait();
-                        HttpClient client = new() { BaseAddress = new Uri("http://localhost:5000/graphql") };
-                        IntrospectionClient introspectionClient = new();
-                        Task<HotChocolate.Language.DocumentNode> response = introspectionClient.DownloadSchemaAsync(client);
-                        response.Wait();
-
-                        HotChocolate.Language.DocumentNode node = response.Result;
-                        File.WriteAllText(Path.Combine(options.OutputDirectory, options.GraphQLSchemaFile), node.ToString());
+                        int retryCount = 5;
+                        int tries = 0;
+
+                        while (tries < retryCount)
+                        {
+                            try
+                            {
+                                HttpClient client = new() { BaseAddress = new Uri("http://localhost:5000/graphql") };
```

Comment:
why not https?

Reply:
Mostly because it's an in-proc local connection so I didn't want to worry about self-signed cert/devserver cert causing problems.

Is that the cause of the static analysis failure?

Reply:
Updated to use HTTPS, but it does mean we have to use a custom `HttpClientHandler` to ignore the SSL errors

Reply:
With the usage of HTTPS and the custom handler to ignore SSL errors, Semmle code analysis fails: 

`##[warning]1. Semmle Warning SM02185 - File: src/Cli/Program.cs. Line: 132. Column 53. 
  Tool: Semmle: Rule: SM02185 (cs/httpclient-checkcertrevlist-disabled). 
  HttpClient is created without enabling certificate revocation.`

Not sure if it was failing previously with `http`. Is there any other alternative?

Reply:
Yes, it was failing previously because it wanted a HTTPS call to be made.

Could we disable that warning for that line?